.github/workflows/ci.yml

```diff
@@ -0,0 +1,153 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - "release/**"
+env:
+  APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+  APPLITOOLS_BATCH_ID: ${{ github.sha }}
+  APPLITOOLS_BATCH_NAME: ${{ github.head_ref }}
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: v12.20.0
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: |
+            **/node_modules
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: lerna prepare
+        if: steps.yarn-cache.outputs.cache-hit == 'true'
+        run: lerna run prepare
+      - name: install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --prefer-offline
+      - name: lint (eslint, stylelint & prettier)
+        run: yarn lint
+      - name: unit tests (jest & tsc)
+        run: yarn test
+  deploy:
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: v12.20.0
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: |
+            **/node_modules
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: lerna prepare
+        if: steps.yarn-cache.outputs.cache-hit == 'true'
+        run: lerna run prepare
+      - name: install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --prefer-offline
+      - name: setup AWS
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: set $SHA7
+        run: echo "SHA7=$(git rev-parse --short ${{ github.event.pull_request.head.sha || github.sha }})" >> $GITHUB_ENV
+      - name: set $URL_STORYBOOK
+        run: echo "URL_STORYBOOK="https://${SHA7}.ods.dev/sb/index.html"" >> $GITHUB_ENV
+      - name: get $COMMIT_MSG
+        run: |
+          echo "COMMIT_MSG<<EOF" >> $GITHUB_ENV
+          git log --format=%B -n 1 ${{ github.event.after }} >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: build docs
+        run: |
+          yarn workspace @okta/odyssey-react build-storybook
```

Comment:
this appears to be missing the legacy doc site build? Was this intentional because it just slows down CI? We would also benefit from skipping the dependency installs for all those legacy packages. Perhaps we can archive them in a feature branch and remove them from `develop` ?


Also - if this work is necessary for VRT below we can explore caching the resulting `storybook-static` folder to skip the need to rebuild the storybook each time CI runs!

Reply:
> this appears to be missing the legacy doc site build? Was this intentional because it just slows down CI? 
Correct!

> We would also benefit from skipping the dependency installs for all those legacy packages. Perhaps we can archive them in a feature branch and remove them from develop?

Great call. I will remove the docs and prism theme packages in a follow up PR!

> Also - if this work is necessary for VRT below we can explore caching the resulting storybook-static folder to skip the need to rebuild the storybook each time CI runs!

This is interesting, didn't consider this to be an option. May be worth spiking in a future sprint!

Reply:
> Great call. I will remove the docs and prism theme packages in a follow up PR!

Can we log this ticket as part of the docs migration effort? I'd prefer to grab all of that content from develop and then remove the package when it's migrated, rather than archive it ahead of time.

Reply:
@edburyenegren-okta ill defer to you on this. My preference is to remove it in a fast follow as it slows down our CI because we have to install all of the associated deps for the two packages -- but I have dealt with this up until now, so I can live with it for a few more weeks 😂

Reply:
@edburyenegren-okta can you explain your reasoning here? We can also bring the code back from another feature branch or revert commit but I want to make sure I understand the why. I would love to have CI faster and our dependency installs simpler right now! It will make day to day life easier for us

Reply:
here is a PR to help with this! https://github.com/okta/odyssey/pull/1087

Reply:
@jeffbelser-okta These are currently the only reference docs we have for implementation, behavior, and a handful of design rules. The published site isn't a safe reference, as we haven't updated the documentation there since Nov 2020.

I use them to sanity check our resulting UI, which aren't yet to parity, and field questions about components that aren't documented in the `0.7.0` release (e.g. Infobox).

Because I use them regularly, I'd rather not have to swap branches to access the files. It also feels like bad process to remove something from `develop` without a solution in place for consumers.

If removing these packages has become high priority, I am happy to take on the migration work next sprint, as slotted in the backlog.